gubernator/filters.py

```diff
@@ -100,8 +106,38 @@ def do_parse_pod_name(text):
     else: 
         return ""
 
+def do_label_attr(labels, name):
```

Comment:
This looks like it's only used for size labels. Might add a comment saying as much or rename, since it's pretty confusing.